cloud/app/server/server.js

```diff
@@ -675,8 +675,8 @@ class _robotAgent extends Capability {
     const devices = {};
     _.each(org, (device, deviceId) => {
       if (deviceId.startsWith('_')) return; // not a device
-      const versions = device['@transitive-robotics']['_robot-agent'];
-      const merged = mergeVersions(versions, field);
+      const versions = device?.['@transitive-robotics']?.['_robot-agent'];
+      const merged = versions ? mergeVersions(versions, field) : {};
```

Comment:
Can you explain why this is needed? How can `device` be undefined? Under what circumstances could the org contain `undefined` values?